stories/core/Table.stories.tsx

```diff
@@ -718,3 +759,119 @@ export const NoData: Story<TableProps> = ({ columns, data, ...rest }) => {
 NoData.args = {
   data: [],
 };
+
+export const ControlledState: Story<TableProps> = (args) => {
+  const { columns, data, ...rest } = args;
+
+  const [selectedRows, setSelectedRows] = useState<Record<string, boolean>>({});
+
+  const onSelect = useCallback((rows, state) => {
+    action(
```

Comment:
This callback should be removed. It's not doing anything to the state, and is just adding noise/confusion.

Reply:
Removed.